frontend/src/components/ReleaseDescription.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components/macro'
+
+export default function ReleaseDescription({release}){
+
+    const renderReleaseYear = () => {
+        if (release.releaseYear === 0){
+            return <></>
+        } else {
+            return <h3>{release.releaseYear}</h3>
+        }
+    }
+
+    return (
+        <Release>
+            <img src={release.coverImageUrl} alt={'Shows the release cover'}/>
+            <section>
+                <h3>{release.fullTitle}</h3>
+                {renderReleaseYear()}
```

Comment:
this way you don't need the renderReleaseYear function

```suggestion
                {release.releaseYear !== 0 &&  <h3>release.releaseYear</h3>}
```

Reply:
thx. changed ...